Make sure easybuild flags and libs are used for llvmlite
Author: Samuel Moors, Vrije Universiteit Brussel (VUB)
diff -ur llvmlite-0.26.0.orig/ffi/Makefile.linux llvmlite-0.26.0/ffi/Makefile.linux
--- llvmlite-0.26.0.orig/ffi/Makefile.linux	2019-02-01 21:16:16.000000000 +0100
+++ llvmlite-0.26.0/ffi/Makefile.linux	2019-02-14 13:38:18.147608713 +0100
@@ -5,9 +5,9 @@
 CXX_FLTO_FLAGS ?= -flto
 LD_FLTO_FLAGS ?= -flto -Wl,--exclude-libs=ALL
 
-CXXFLAGS = $(LLVM_CXXFLAGS) $(CXX_FLTO_FLAGS)
+CXXFLAGS := $(CXXFLAGS) $(LLVM_CXXFLAGS) $(CXX_FLTO_FLAGS)
 LDFLAGS := $(LDFLAGS) $(LLVM_LDFLAGS) $(LD_FLTO_FLAGS)
-LIBS = $(LLVM_LIBS)
+LIBS := $(LIBS) $(LLVM_LIBS)
 INCLUDE = core.h
 SRC = assembly.cpp bitcode.cpp core.cpp initfini.cpp module.cpp value.cpp \
 	  executionengine.cpp transforms.cpp passmanagers.cpp targets.cpp dylib.cpp \
